src/components/roomHeader/RoomHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import { MainContainer, IndexText, TitleText } from './RoomHeader.styles';
+interface RoomHeader {
+  index: string;
+  title: string;
+}
+
+const RoomHeader = (props: RoomHeader) => {
+  return (
+    <MainContainer>
+      <IndexText>{props.index}</IndexText>
+      <TitleText>{props.title}</TitleText>
+    </MainContainer>
+  );
+};
```

Comment:
`c`: [비구조화 할당](https://learnjs.vlpt.us/useful/06-destructuring.html) 문법으로 쓰시면 조금 더 간결하게 쓸 수 있을 것 같습니다!
```suggestion
const RoomHeader = ({ index, title }: RoomHeader) => {
  return (
    <MainContainer>
      <IndexText>{index}</IndexText>
      <TitleText>{title}</TitleText>
    </MainContainer>
  );
};
```

Reply:
적용했습니다 감사합니다 :)